### DISCUSSION TITLE
NOOB to EpoxyDuino maybe using SubModules

### DISCUSSION THREAD

**Discussion by MageMCU** [bxparks/UnixHostDuino#37] [General]
I am interested in using your repository but I am a NOOB to Git, GitHub and incorporating others software from GitHub without stepping on toes. Any advice thank you.

Have you ever work with SubModules? Is there a better alternative? Thank you.

Source: GitHub Topic: https://github.blog/2016-02-01-working-with-submodules/

MageMCU

**Comment by bxparks** [comment]
If you have specific questions about EpoxyDuino, this is the place for it. But you are not going to win many favors by asking random project maintainers to teach you how to use git, GitHub and git submodules. There is an entire internet at your finger tips. I understand that everyone starts off as a noob, and some places on the internet are inappropriately hostile to beginners (e.g. the Arduino Forums). For what it's worth, I don't use submodules, and there are hundreds of blog posts and articles on the internet explaining why people don't like it. Good luck on your research.
